Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -147,6 +147,44 @@ function Add-User($authUrl, $name, $accessToken) {
     return Invoke-Post $url $body $accessToken
 }
 
+function Add-UserOrGroupToEdwAdmin($userOrGroup, $connString) {
+	$query   = "DECLARE @userId INTEGER;
```

Comment:
why is this here as well as utilities?  
I must be missing something?

Reply:
So if you look, there are two files with the same functions.  That is because one is a Utilities powershell script and the other is this, which is the older powershell functions.  Anyways, as I understand the older powershell script, this one, is depreciated and will be removed eventually and replaced by the utility script and the quiet install.

So if it is going to be removed soon, do we want to reuse the code or just let the old code die?

Reply:
After this change, I think we can let the old code die. No need to do additional work to include the utilities module here.